eve/workers/pod-linter/pod.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "lint-worker"
+spec:
+  containers:
+  - name: lint-container
+    image: {{ images['docker-linter'] }}
+    resources:
+      requests:
+        cpu: 500m
+        memory: 1Gi
+      limits:
+        cpu: "1"
+        memory: 2Gi
+    securityContext:
+      capabilities:
+        drop:
+        - ALL
+      runAsUSer: 1000
```

Comment:
we have set up eve as the internal user no ? and the capabilities isnt it  by default dropped ?

Reply:
No idea, that's straight from the metal 1.3 pod file. I'll take a closer look at it.

Reply:
This is because running things as root is ugly. Yes the image setup  the eve-worker manually to use 1000 user (https://github.com/scality/metalk8s/blob/development/1.3/eve/images/pep8-checker/Dockerfile#L13). You can't pass username (just uid) in kubernetes yaml, as far as I know.